### ISSUE TITLE
Can't see names of folders in menu window using Dynamic theme

### ISSUE THREAD

**Issue by clhendricksbc** [standardnotes/forum#1554]
I'm using the MacOS app (3.12.1), and after a recent update the "folders" section of the left menu is pushed to the right so far that I can't see the full names of my folders, nor the names of nested folders (for those, I can only see the first letter of the folder name). This is when I hover the cursor over the left menu--the notes window won't move to the right enough for me to see my folder names. I have tried to find out ways to change this behaviour through searching documentation but I can't seem to do so. Sorry if I'm missing something there that will address it!

Right now this is making SN practically unusable for me, because I have to wait for the hover tool tip to tell me which note I'm clicking on in the left menu.

![Screen Shot 2022-03-07 at 10 34 36 AM](https://user-images.githubusercontent.com/4546117/157096989-e353cac7-f149-4e1c-b35a-1f39b1c278e4.png)

**Comment by atmoio** [comment]
You should be able to hover over the right side of the tags panel, then drag to resize so that the names fit into view 🙂

**Comment by clhendricksbc** [comment]
Thank you for responding so quickly! I tried it and it didn't let me resize. When I tred to drag tags panel window to the right it wouldn't go any further.

However, I just figured out the issue. I remembered that at one point a long time ago I installed an extension that let the size of the panel change to smaller when not in use and larger when in use. The "larger" version is what is shown in the screen shot above, and the "smaller" version just shows the hashtags. I went into preferences and figured out which extension that is ("Dynamic") and uninstalled it. That fixed the problem.

However, I think this may mean that the Dynamic extension is not really usable with the new tags/folder setup that happened over the last couple of months or so (not sure which release). Possibly that's an old extension anyway that few are using anymore...and I'm just behind! 😄
